contracts/src/periphery/cert/interfaces/IEigenDACertVerifier.sol

```diff
@@ -1,14 +1,9 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.9;
 
-interface IEigenDACertVerifier {
-    /// @notice Check a DA cert's validity, and revert if invalid.
-    function verifyDACert(bytes calldata certBytes) external view;
-
-    /// @notice Check a DA cert's validity
-    /// @return status An enum value. Success is always mapped to 1, and other values are errors specific to each CertVerifier.
-    function checkDACert(bytes calldata certBytes) external view returns (uint8 status);
+import {IEigenDACertVerifierBase} from "src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol";
 
+interface IEigenDACertVerifier is IEigenDACertVerifierBase {
     /// @notice Returns the certificate version. Used off-chain to identify how to encode a certificate for this CertVerifier.
     function certVersion() external view returns (uint8);
```

Comment:
let's use a broader type like uint64? You never know when we'll need the flexibility, either for more certVersions or for other encodings where we want to use the extra bytes to mean something (say encode a semver).

Reply:
Sure, it can be anything as long as it doesn't change later! Maybe even just a uint256 if u want.